src/lister_haxe/test/lister_test.hx

```diff
@@ -0,0 +1,36 @@
+package test;
+
+import haxe.unit.TestCase;
+using StringTools;
+using test.TestCaseExtender;
+
+class Lister_Test extends TestCase
+{
+    // It ensures returned urls are fully qualified
+    @:access(Lister)
```

Comment:
I like how this syntax works for accessing private functions from a test class, as opposed to `friend` or `protected internal` or whatever on the class-under-test.  Curious to see how this compiles to C#.